src/analyzers/dingo/checks/helper/get_configureMethods.go

```diff
@@ -0,0 +1,45 @@
+package helper
+
+import (
+	"go/ast"
+	"reflect"
+
+	"golang.org/x/tools/go/analysis"
+	"golang.org/x/tools/go/analysis/passes/inspect"
+	"golang.org/x/tools/go/ast/inspector"
+)
+
+// helper analyzer which returns all function declarations
+var ConfigureDeclAnalyzer = &analysis.Analyzer{
```

Comment:
Not so happy that this is listed within the checks then. But for now it ts ok i think.